pyproject.toml

```diff
@@ -60,7 +62,9 @@ docs = [
     "sphinx-rtd-theme",
     "sphinxcontrib-apidoc",
     "sphinxext.remoteliteralinclude",
+    "myst-parser"
```

Comment:
Is this for moving to markdown?

Reply:
That is the plane. But it will be in a different PR. I should revert the changes.

Reply:
No need to revert.